tests/webExtension/test_web_extension.py

```diff
@@ -95,6 +95,42 @@ async def test_extensions_can_install(websocket, unpacked_extension_location,
     assert result['extension']
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize('capabilities', [{
+    'goog:chromeOptions': {
+        'args':
+            ['--enable-unsafe-extension-debugging', '--remote-debugging-pipe']
+    },
+}],
+                         indirect=True)
+async def test_extensions_can_open_manifest(websocket,
+                                            unpacked_extension_location,
+                                            test_headless_mode,
+                                            create_context):
+    """ Verify the extension page can be opened. http://b/412926721 """
```

Comment:
Let's use crbugs when the issue is on the Chromium tracker.
```suggestion
    """ Verify the extension page can be opened. https://crbug.com/412926721 """
```